cuML/src/glm/glm.cu

```diff
@@ -29,14 +29,17 @@ void olsFit(float *input, int n_rows, int n_cols, float *labels, float *coef,
 	CUBLAS_CHECK(cublasCreate(&cublas_handle));
 
 	cusolverDnHandle_t cusolver_handle = NULL;
-	CUSOLVER_CHECK(cusolverDnCreate(&cusolver_handle));
+  CUSOLVER_CHECK(cusolverDnCreate(&cusolver_handle));
```

Comment:
I think all of these mis-alignments are a result of inconsistent formatting in these files. People using tabs and not converting them to spaces.

Reply:
Thinking out loud here, but it would not be a bad idea for us to export an eclipse xml formatter that we could check in to our repository so that contributors to cuML (and perhaps the larger RAPIDS ecosystem) can all be using the same formatting. 

For now, I would reformat the files just to make sure they are consistent.

Reply:
Reformatted all the files marked in this review to use tabs.